pack/return/rttfa_encounter.json

```diff
@@ -22,5 +462,739 @@
+		"clues": 0,
+		"code": "53041",
+		"double_sided": true,
+		"encounter_code": "return_to_the_boundary_beyond",
+		"encounter_position": 4,
+		"faction_code": "mythos",
+		"illustrator": "Greg Bobrowski",
+		"name": "Chapultepec Park",
+		"pack_code": "rttfa",
+		"position": 41,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - After you fail an [agility] test while at Chapultepec Park: Take 1 damage.\n[action]: <b>Explore.</b> Test [agility] (3). If you succeed, draw the top card of the exploration deck. If it is a location with a matching location symbol in the upper-left corner, place it on top of this location, taking its place.",
+		"traits": "Mexico City. Present-Day.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "The main square in central Mexico City, once the ceremonial center of Tenochtitlán, stretches before you. Silver lightning flashes in the storm clouds overhead, and you are shocked to find a tide of blood swelling over the plaza. The entire city is bizarrely devoid of any life.",
+		"clues": 0,
+		"code": "53042",
+		"double_sided": true,
+		"encounter_code": "return_to_the_boundary_beyond",
+		"encounter_position": 5,
+		"faction_code": "mythos",
+		"illustrator": "Vlad Ricean",
+		"name": "Zócalo",
+		"pack_code": "rttfa",
+		"position": 42,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "[action] Discard cards with a combined total of of at least 4 skill icons from your hand: <b>Explore.</b> Draw the top card of the exploration deck. If it is a location with a matching location symbol in the upper-left corner, place it on top of this location, taking its place.",
+		"traits": "Mexico City. Present-Day.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "A small wooden boat bumps up against the side of the canal, swaying gently in the rippling water. A juniper tree sticks out of the shallow water nearby, bending in the mild breeze.",
+		"clues": 0,
+		"code": "53043",
+		"double_sided": true,
+		"encounter_code": "return_to_the_boundary_beyond",
+		"encounter_position": 6,
+		"faction_code": "mythos",
+		"illustrator": "Vlad Ricean",
+		"name": "Xochimilco",
+		"pack_code": "rttfa",
+		"position": 43,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "[reaction] After you successfully fight or evade an enemy at Xochimilco: <b>Explore.</b> Draw the top card of the exploration deck. If it is a location with a matching location symbol in the upper-left corner, place it on top of this location, taking its place.",
+		"traits": "Mexico City. Present-Day.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "Although not always a part of Mexico City, Coyoacán has always been a historic area, marked by narrow cobblestone streets and plazas. The ground shakes as you walk down one of the village’s many confined paths, and you suspect there is more that lives below the surface…",
```

Comment:
Yeah, I think I grabbed this from the original. The weird presentation here is likely because github displays the text as a monospace font (each character the same width), so unicode characters get squeezed even though they'll render normally on the site.